### DISCUSSION TITLE
Sidebar doesn't scroll to current page

### DISCUSSION THREAD

**Discussion by thibaultmol** [mkdocs/mkdocs#2854] [Q&A]
Hi,

I was looking at TrueChart's mkdocs instance. I'm not sure if this is a bug/missing feature/configuration error.
If I go to this page
https://truecharts.org/apps/stable/nextcloud/
I would expect the list on the left to scroll to 'nextcloud' so you can see it's subitems. 
But if I load that page, it just shows the top of that list instead.

What I see:
![Screenshot_20220523_205301](https://user-images.githubusercontent.com/1010226/169887226-035859d1-18d4-4bd3-bc2e-f6ef377ab027.png)

What I would expect:
![Screenshot_20220523_205315](https://user-images.githubusercontent.com/1010226/169887230-7d735b6b-b994-435d-b4a7-7e8122183908.png)

https://github.com/truecharts/pub/issues/38

**Comment by squidfunk** [comment]
Thanks for asking. This is essentially a Material for MkDocs question. The feature you're asking for was originally reported in https://github.com/squidfunk/mkdocs-material/issues/2155 and has been implemented as part of [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) and released 3.5 months ago.
